### DISCUSSION TITLE
Add new configuration files for Swin 新版 config 适配——Swin

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#542] [MMAction2]
# Add new configuration files for Swin 
 新版 config 适配——Swin 

mmengine implements a new config mechanism, based on the new config mechanism, please update configuration file for adaptation. Target config path: configs/recognition/swin

mmengine 实现了一套新的 config 机制，基于新的 config 机制进行配置文件的适配
- 相关适配教程文档：[https://aicarrier.feishu.cn/docx/PkICdaJbpoOVjLxMFi2cYqPjnOc](https://aicarrier.feishu.cn/docx/PkICdaJbpoOVjLxMFi2cYqPjnOc)  
该任务适配的 config 目录：configs/pv_rcnn/ ，完成所有文件适配

## Task Information

**Repository**：MMAction2

**Difficulty**：Easy

**Points**：10

**Estimated Time Consumption**：7 days

## Task Status

**Claimant's Github ID**：ZhaoCake

**Claim Time**：2023年08月14日

**PR Link**：https://github.com/open-mmlab/mmaction2/pull/2649

## Points to Note

❗️请注意：此任务属于超级视客营任务，不能被领取！请登陆 [超级视客营平台](https://openmmlab.com/activity/codecamp) 按照流程申请任务，我们会尽快审核～

**Comment by ZhaoCake** [comment]
https://github.com/open-mmlab/mmaction2/pull/2649
